pkg/cli/admin/upgrade/recommend/alerts.go

```diff
@@ -0,0 +1,185 @@
+				// ideally the upstream PodDisruptionBudget*Limit alerts templated in the namespace and PDB, but until they do, inject those ourselves
+				details = fmt.Sprintf("Namespace=%s, PodDisruptionBudget=%s. %s", alert.Labels.Namespace, alert.Labels.PodDisruptionBudget, details)
+			}
+			conditions = append(conditions, metav1.Condition{
+				Type:    fmt.Sprintf("recommended/CriticalAlerts/%s/%d", alertName, i),
+				Status:  metav1.ConditionFalse,
+				Reason:  fmt.Sprintf("Alert:%s", alert.State),
+				Message: fmt.Sprintf("%s alert %s %s, suggesting significant cluster issues worth investigating. %s", alert.Labels.Severity, alert.Labels.AlertName, alert.State, details),
+			})
+			i += 1
+			continue
+		}
+
+		if alertName == "PodDisruptionBudgetAtLimit" {
+			havePodDisruptionBudget = true
+			// ideally the upstream PodDisruptionBudget*Limit alerts templated in the namespace and PDB, but until they do, inject those ourselves
+			details = fmt.Sprintf("Namespace=%s, PodDisruptionBudget=%s. %s", alert.Labels.Namespace, alert.Labels.PodDisruptionBudget, details)
+			conditions = append(conditions, metav1.Condition{
+				Type:    fmt.Sprintf("recommended/PodDisruptionBudgetAlerts/%s/%d", alertName, i),
+				Status:  metav1.ConditionFalse,
+				Reason:  fmt.Sprintf("Alert:%s", alert.State),
+				Message: fmt.Sprintf("%s alert %s %s, which might slow node drains. %s", alert.Labels.Severity, alert.Labels.AlertName, alert.State, details),
+			})
+			i += 1
+			continue
+		}
+
+		if alertName == "KubeContainerWaiting" {
+			havePullWaiting = true
+			conditions = append(conditions, metav1.Condition{
+				Type:    fmt.Sprintf("recommended/PodImagePullAlerts/%s/%d", alertName, i),
+				Status:  metav1.ConditionFalse,
+				Reason:  fmt.Sprintf("Alert:%s", alert.State),
+				Message: fmt.Sprintf("%s alert %s %s, which may slow workload redistribution during rolling node updates. %s", alert.Labels.Severity, alert.Labels.AlertName, alert.State, details),
+			})
+			i += 1
+			continue
+		}
+
+		if alertName == "KubeNodeNotReady" || alertName == "KubeNodeReadinessFlapping" || alertName == "KubeNodeUnreachable" {
+			haveNodes = true
+			conditions = append(conditions, metav1.Condition{
+				Type:    fmt.Sprintf("recommended/NodeAlerts/%s/%d", alertName, i),
+				Status:  metav1.ConditionFalse,
+				Reason:  fmt.Sprintf("Alert:%s", alert.State),
+				Message: fmt.Sprintf("%s alert %s %s, which may slow workload redistribution during rolling node updates. %s", alert.Labels.Severity, alert.Labels.AlertName, alert.State, details),
+			})
+			i += 1
+			continue
+		}
+	}
+
+	if !haveCritical {
+		conditions = append(conditions, metav1.Condition{
+			Type:    "recommended/CriticalAlerts",
+			Status:  metav1.ConditionTrue,
+			Reason:  "AsExpected",
+			Message: "No critical alerts firing.",
+		})
+	}
+
+	if !havePodDisruptionBudget {
+		conditions = append(conditions, metav1.Condition{
+			Type:    "recommended/PodDisriptionBudgetAlerts",
```

Comment:
```suggestion
			Type:    "recommended/PodDisruptionBudgetAlerts",
```

Reply:
thanks!  Fixed as part of the 426959c86 -> 4d19abdf7 reroll.